test/common.js

```diff
@@ -0,0 +1,36 @@
+import co  from 'co';
+import jwt from 'jsonwebtoken';
+
+import config from '../src/config';
+
+const endpointPrefix = `/${config.get('version')}`;
+
+export function loginAsAdmin(server) {
+  return co(function*() {
```

Comment:
if you're not familiar with `co`, it returns a Promise and takes a generator as argument; this makes it easier to deal with Promise-returning functions. `yield` will wait for the Promise to be resolved, rejection translates to exceptions, etc. This is so good :)